src/entity/units/glop.ts

```diff
@@ -18,11 +18,14 @@ const unit: UnitSource = {
     subtype: UnitSubType.RANGED_RADIUS,
   },
   unitProps: {
+    damage: 20,
     attackRange: 400,
-    bloodColor: bloodLobber,
     healthMax: 60,
-    damage: 20,
-    manaCostToCast: 15
+    mana: 30,
+    manaMax: 30,
+    manaPerTurn: 10,
+    manaCostToCast: 5,
```

Comment:
5 cost with 10 regen means the glop will never run out of mana on it's own.  Could be fine, just wondering why?

Reply:
The glop is the first/most frequent mana enemy you see, so I sort've did this as a way to make mana burn/steal a little stronger. Both the regen and cost amounts felt right thematically / when compared to other units and their abilities.

Reply:
Works for me!